src/vendor/btl_bloomfilter/pythonInterface/python_wrapper.readme

```diff
@@ -0,0 +1,21 @@
+Python wrapper for BloomFilter.hpp
```

Comment:
Do we need to be including all of these interface files, etc. in this commit?

Reply:
The intention was to just unzip the vendored repo and forget about it. Currently, abyss is like that too

Reply:
Ok makes sense!